scss/components/_off-canvas.scss

```diff
@@ -118,13 +138,24 @@ $maincontent-class: 'off-canvas-content' !default;
     position: absolute;
   }
 
+  // Set the off-canvas z-index.
   z-index: $offcanvas-push-zindex;
 
+  // Increase CSS specificity
```

Comment:
I've done this to override the fix z-index that is set in https://github.com/zurb/foundation-docs/blob/master/scss/_page.scss
If this gets removed we'll be able to undo this specificity fix.